### PR TITLE
Add ability to enable TF debug mode

### PR DESCRIPTION
**Pull request by igorkotof**
[ICMK] Added ability to enable TF debug mode

How to use:
- Add `TF_LOG_LEVEL=TRACE` var to Makefile 
- or
- to .infra/env/<$ENV>/env.mk file

### CODE REVIEW

**Comment by igorkotof** [comment]
[[ICMK] to add ability to enable any debug mode within terraform (TF_LOG)](https://trello.com/c/5KNeqbs6/215-icmk-to-add-ability-to-enable-any-debug-mode-within-terraform-tflog)

File: terraform/main.mk [hunk 1]
```diff
@@ -37,7 +37,9 @@ TERRAFORM_STATE_BUCKET_NAME ?= $(NAMESPACE)-tf-state
 CHECKOV ?= $(DOCKER) run -v $(ENV_DIR):/tf -i bridgecrew/checkov -d /tf -s
 TFLINT ?= $(DOCKER) run --rm -v $(ENV_DIR):/data -t wata727/tflint
 TFLOCK ?= $(DOCKER) run --rm --hostname=$(USER)-icmk-terraform -v $(ENV_DIR):/$(ENV_DIR) -v "$(ENV_DIR)/.terraform":/"$(ENV_DIR)/.terraform" -v "$(INFRA_DIR)":"$(INFRA_DIR)" -v $(HOME)/.aws/:/root/.aws:ro -w $(ENV_DIR) -e AWS_PROFILE=$(AWS_PROFILE) -e ENV=$(ENV) hazelops/tflock
-TERRAFORM ?= $(DOCKER) run --rm --hostname=$(USER)-icmk-terraform -v $(ENV_DIR):/$(ENV_DIR) -v "$(ENV_DIR)/.terraform":/"$(ENV_DIR)/.terraform" -v "$(INFRA_DIR)":"$(INFRA_DIR)" -v $(HOME)/.aws/:/root/.aws:ro -w $(ENV_DIR) -e AWS_PROFILE=$(AWS_PROFILE) -e ENV=$(ENV) hashicorp/terraform:$(TERRAFORM_VERSION)
+TF_LOG_LEVEL ?=
```
Comment (gloovers) [inline terraform/main.mk:40]: @igorkotof why don't we use the same variable name as terraform has `TF_LOG`

Comment (igorkotof) [reply terraform/main.mk:40]: @gloovers as for me this name is most defining essence, agree?

Comment (igorkotof) [reply terraform/main.mk:40]: or we can name it like TF_LOG_VERBOSITY

**Review by gloovers — APPROVED** [review]
👍

**Pull request merged**